### PR TITLE
feat(#101): implemented exclusion by author

### PR DESCRIPTION
**Pull request by l3r8yJ**
@h1alexbel take a look, please

---

## PR-Codex overview
This PR focuses on enhancing skip logic in code reviews by introducing `SkipAuthors` and `SkipIfMentioned`. 

### Detailed summary
- Added `SkipAuthors` to handle authors exclusion
- Updated `SkipIfMentioned` to use `SkipAuthors` for skipping logic
- Added integration test for `SkipAuthors`
- Added integration test for `SkipIfMentioned`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -139,6 +139,12 @@ SOFTWARE.
       <artifactId>logback-classic</artifactId>
       <version>${logback-classic.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.junit-pioneer</groupId>
+      <artifactId>junit-pioneer</artifactId>
+      <version>2.2.0</version>
```
Comment (h1alexbel) [inline pom.xml:145]: let's wrap it into `${junit-pioneer.version}`

**Review by h1alexbel — COMMENTED** [review]
@l3r8yJ one comment

**Comment by h1alexbel** [comment]
@l3r8yJ yeah, what kind of integration test?

**Comment by l3r8yJ** [comment]
> @l3r8yJ yeah, what kind of integration test?

@h1alexbel I think this test have to check pull request from author from `skip_authors`

**Comment by h1alexbel** [comment]
@l3r8yJ got it, could you create puzzle for this one?

**Comment by l3r8yJ** [comment]
@h1alexbel sure

**Comment by h1alexbel** [comment]
@l3r8yJ also I thought it would be useful to create `tracehubpm/test` repo for integration testing, instead of my own which is private. So you and other contributors can create new integration tests

**Comment by l3r8yJ** [comment]
> @l3r8yJ also I thought it would be useful to create `tracehubpm/test` repo for integration testing, instead of my own which is private. So you and other contributors can create new integration tests

@h1alexbel ill add this puzzle too

**Comment by l3r8yJ** [comment]
@h1alexbel added

**Comment by h1alexbel** [comment]
@l3r8yJ thanks

**Pull request merged**